ads/google/a4a/utils.js

```diff
@@ -130,6 +136,9 @@ function buildAdUrl(
   const iframeDepth = iframeNestingDepth(global);
   const dtdParam = {name: 'dtd'};
   const adElement = a4a.element;
+  if (ValidAdContainerTypes.indexOf(adElement.parentElement.tagName) >= 0) {
```

Comment:
Linear search!  Ick!  Well, okay, I guess it's just on a list of length two...  :-)